packet-simplemessage.lua

```diff
@@ -284,20 +323,24 @@ do
 		[MOTO_MOTION_REPLY_SUBCODE_INVALID_DATA_POSITION    ] = "Data Position",
 		[MOTO_MOTION_REPLY_SUBCODE_INVALID_DATA_SPEED       ] = "Data Speed",
 		[MOTO_MOTION_REPLY_SUBCODE_INVALID_DATA_ACCEL       ] = "Data Acceleration",
-		[MOTO_MOTION_REPLY_SUBCODE_INVALID_DATA_INSUFFICIENT] = "Data Insufficient"
+		[MOTO_MOTION_REPLY_SUBCODE_INVALID_DATA_INSUFFICIENT] = "Data Insufficient",
+		[MOTO_MOTION_REPLY_SUBCODE_INVALID_DATA_TIME        ] = "Data Time",
+		[MOTO_MOTION_REPLY_SUBCODE_INVALID_DATA_TOOLNO      ] = "Data Tool Number",
 	}
 
 	local motoman_not_read_code_str = {
-		[MOTO_MOTION_REPLY_NOTREADY_CODE_UNSPECIFIED] = "Unspecified",
-		[MOTO_MOTION_REPLY_NOTREADY_CODE_ALARM      ] = "Alarm",
-		[MOTO_MOTION_REPLY_NOTREADY_CODE_ERROR      ] = "Error",
-		[MOTO_MOTION_REPLY_NOTREADY_CODE_ESTOP      ] = "E-Stop",
-		[MOTO_MOTION_REPLY_NOTREADY_CODE_NOT_PLAY   ] = "Not Play",
-		[MOTO_MOTION_REPLY_NOTREADY_CODE_NOT_REMOTE ] = "Not Remote",
-		[MOTO_MOTION_REPLY_NOTREADY_CODE_SERVO_OFF  ] = "Servo Off",
-		[MOTO_MOTION_REPLY_NOTREADY_CODE_HOLD       ] = "Hold",
-		[MOTO_MOTION_REPLY_NOTREADY_CODE_NOT_STARTED] = "Not Started",
-		[MOTO_MOTION_REPLY_NOTREADY_CODE_WAITING_ROS] = "Waiting ROS"
```

Comment:
@gavanderhoorn Did you remove the string for `MOTO_MOTION_REPLY_NOTREADY_CODE_WAITING_ROS` on purpose?

Reply:
No.

Looks like a copy-pasta mistake.